tests/utils/retrieve_asset.rs

```diff
@@ -102,25 +90,18 @@ mod passing {
 
 #[cfg(test)]
 mod failing {
-    use reqwest::blocking::Client;
     use reqwest::Url;
-    use std::collections::HashMap;
 
     use monolith::opts::Options;
     use monolith::utils;
 
     #[test]
     fn read_local_file_with_data_url_parent() {
```

Comment:
These two tests fail when I run `cargo test`.

```shell
test utils::retrieve_asset::passing::read_data_url ... ok
test utils::retrieve_asset::passing::read_local_file_with_file_url_parent ... ok
error: The following required arguments were not provided:
    <target>

USAGE:
    mod-f99b3136656731cd [OPTIONS] <target>

For more information try --help
error: test failed, to rerun pass `--test mod`

Caused by:
  process didn't exit successfully: `/home/[...]/dev/monolith/target/debug/deps/mod-f99b3136656731cd` (exit status: 2)
```

 I think this is a consequence of the HTTP client initialization changes. When `retrieve_asset` is called, `CLIENT` setup in turn references `OPTIONS`, which calls `Options::from_args()` even though this isn't a CLI context. The two tests above in the `passing` module execute code paths that don't touch `CLIENT` and thereby avoid this.